integration_tests/src/main/python/conftest.py

```diff
@@ -102,6 +102,12 @@ def skip_unless_precommit_tests(description):
 
 _limit = -1
 
+_inject_oom = None
+
+def should_inject_oom():
+    global _inject_oom
```

Comment:
redundant line?
```suggestion
```

Reply:
I'll take care of this in a future PR, but wasn't going to kick the CI again since it passed and its been waiting for a while. I'd like to start running all PRs with the change.